src/java/com/puppetlabs/http/client/impl/PersistentSyncHttpClient.java

```diff
@@ -26,10 +31,42 @@ private static void logAndRethrow(String msg, Throwable t) {
         throw new HttpClientException(msg, t);
     }
 
+    public Map<String, Timer> getClientMetrics(){
+        return Metrics.getClientMetrics(metricRegistry);
+    }
+
+    public Map<String, Timer> getClientMetrics(String url, Metrics.MetricType metricType) {
```

Comment:
I'm not too fussed about this either way, but it did seem easier to me for the users to not need to worry about holding references to both the `MetricsRegistry` and the client object, and correlating the two of them.  Seems easier to me for them to just grab a temporary local reference to the metrics registry from tk-metrics at the time when they're constructing the client, pass it into the constructor, and then not have to worry about it anymore afterward.